src/components/Login/Login.scss

```diff
@@ -0,0 +1,48 @@
+.login-container {
```

Comment:
This all looks great. We just want to make sure to alphabetize properties before submitting our project (I have an atom package that can do it automatically if you want me to just check that at the end)